### PR TITLE
add some file checks for KType-Standard.json and kll.json -jt

### PR DESCRIPTION
**Pull request by johanntang**
KType-Standard.json and kll.json errors out if it doesn't exist.  Added a basic check.

### CODE REVIEW

File: kiianigen.js [hunk 1]
```diff
@@ -136,8 +138,11 @@ function main() {
     }
 
     // Get info about the current configuration files from KType-Standard directory.
-    json = JSON.parse(require('fs')
-                          .readFileSync(srcConfigDir + '/KType-Standard.json', 'utf8'));
+    if(!fs.existsSync(srcConfigDir + KTYPE_FILE)) {
+        console.log("KType-Standard.json not found")
```
Comment (intafon) [inline kiianigen.js:142]: Should this kill the script at this point in addition to the log statement?

File: kiianigen.js [hunk 2]
```diff
@@ -162,7 +167,12 @@ function main() {
     }
 
     // Get the max pixel rows and columns
-    kll = JSON.parse(require('fs').readFileSync(srcConfigDir + '/kll.json', 'utf8'));
+
+    if (!fs.existsSync(srcConfigDir + KLL_FILE)) {
+      console.log("kll.json not found")
```
Comment (intafon) [inline kiianigen.js:172]: Ditto on this one...

**Review by intafon — COMMENTED** [review]
Thanks for the changes -- just a couple of questions...

File: kiianigen.js (hunk 1 above)
Comment (johanntang) [reply kiianigen.js:142]: it probably should, but this was the narrowest pull request just to emit a more useful error then the stack trace.

**Pull request merged**